test_flask_static.py

```diff
@@ -182,6 +189,7 @@ def setUp(self):
         self.app.config['FLASKS3_BUCKET_NAME'] = 'foo'
         self.app.config['FLASKS3_USE_CACHE_CONTROL'] = True
         self.app.config['FLASKS3_CACHE_CONTROL'] = 'cache instruction'
+        self.app.config['S3_CACHE_CONTROL'] = '3600'
```

Comment:
Once you set `FLASKS3_USE_CACHE_CONTROL` and `FLASKS3_CACHE_CONTROL`, `S3_CACHE_CONTROL` is used:
https://github.com/e-dard/flask-s3/blob/120339d1401eb45709f7505a0c2321f38cc5d807/flask_s3.py#L490

The other test classes work while creating the app because `FLASKS3_USE_CACHE_CONTROL` default to False.